packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml

```diff
@@ -1,3 +1,3 @@
-url: https://github.com/rancher/aks-operator/releases/download/v1.1.2/rancher-aks-operator-crd-1.1.2.tgz
+url: https://github.com/rancher/aks-operator/releases/download/v1.2.0-rc1/rancher-aks-operator-crd-1.2.0-rc1.tgz
 version: 103.0.0
-doNotRelease: true
```

Comment:
In the next rc or unrc update of chart. Remove this `doNotRelease:false` line

Reply:
@rohitsakala it is removed already, you meant it should be added back next time?

Reply:
It is still there https://github.com/rancher/charts/blob/dev-v2.8/packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml#L3. I meant remove it